### PR TITLE
ETNet

### PR DESCRIPTION
**Pull request by CharlesAuthier**

### CODE REVIEW

File: article/_posts/2019-10-31-ETNet.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    venue:   "Accepted by NeurIPS 2019"
+pdf: "https://arxiv.org/pdf/1910.12056.pdf"
+---
+
+Code available [Here](https://github.com/zhijieW94/ETNet)
+
+
+# Highlights
+
+- The paper introduce the concept of error-correction mechanism to style transfer by evaluating errors in stylization results and correcting them iteratively.
```
Comment (AntoineTheb) [inline article/_posts/2019-10-31-ETNet.md]: the paper ~introduce~ introduces [...] ~style transfer~ style-transfer,

File: article/_posts/2019-10-31-ETNet.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+- Computing the features for perceptual loss in a feed-forward network, each refinement is formulated as an error diffusion process.
+- The overall pyramid-based style transfer framework for style transfer and synthesize detailed results with favored styles.
```
Comment (AntoineTheb) [inline article/_posts/2019-10-31-ETNet.md]: ~synthesize~ synthesized ?

File: article/_posts/2019-10-31-ETNet.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+![](/article/images/ETNet/figure1.png)
+
+# Introduction
+In this paper, they focus on how to transfer arbitrary styles with one single model.
+Even with their current success, the current methods still suffer from artifacts, such as distortions in the semantic structures and detailed style patterns.
+This is because, when there is large variation between content and style images, it is difficult to use the network to synthesize the
```
Comment (AntoineTheb) [inline article/_posts/2019-10-31-ETNet.md]: there is _a_ large

File: article/_posts/2019-10-31-ETNet.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+stylized outputs in a single step for example Figure 1(a).
+
+To solve those artifacts they propose an iterative error-correction mechanism to break the stylization process into multiple refinements like in Figure 1(b).
+Given an insufficiently stylized image and computing what is wrong with the current estimate to transit the error information to the whole image.
+
+The detected errors evaluate the residuals to the ground truth, which thus can guide the refinement effectively.
+Both the stylized images and error features encode information from the same content-style image pairs.
+Thus, a novel error transition module is introduced to predict a residual image that is compatible to the intermediate stylized image.
```
Comment (AntoineTheb) [inline article/_posts/2019-10-31-ETNet.md]: that is compatible ~to~ with

File: article/_posts/2019-10-31-ETNet.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+Compute the correlation between the error features and the deep features extracted from the
+stylized image.
+Then errors are diffused to the whole image according to their similarities.
+
+# Method
+The Figure 2 show the overall framework implemented using a Laplacian pyramid, which has 3 levels.
```
Comment (AntoineTheb) [inline article/_posts/2019-10-31-ETNet.md]: ~The~ Figure _shows_

File: article/_posts/2019-10-31-ETNet.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+Each refinement is performed by an error transition process with an affinity matrix between pixels followed by an error propagation procedure in a coarse-to-fine manner to compute a residual image.
+Each element of the affinity matrix is computed on the similarity between the error feature of one pixel and the feature of stylized result at another pixel, which can be used to measure the possibility of the error transition between them.
+
+![](/article/images/ETNet/figure2.png)
+
+![](/article/images/ETNet/figure3.png)
+
+The Error Transition Network have to generate a residual image for each level of a pyramid.
```
Comment (AntoineTheb) [inline article/_posts/2019-10-31-ETNet.md]: ~have~ has (?)

File: article/_posts/2019-10-31-ETNet.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+
+## Qualitative Comparison
+Figure 6 presents the qualitative comparison results to state of the art methods.
+
+![](/article/images/ETNet/figure6.png)
+
+The output style patterns are more faithful to the target style image, without distortion and exhibiting superior visual detail.
+Meanwhile, this model better respects the semantic structures in the content images, making the style pattern be adapted to these structures.
+
+The Figure 7 show close-up views of transferred results to indicate the superiority in generating style details.
```
Comment (AntoineTheb) [inline article/_posts/2019-10-31-ETNet.md]: ~The~ Figure

**Comment by AntoineTheb** [comment]
@lemairecarl I think you went a little fast here

**Comment by lemairecarl** [comment]
@AntoineTheb oops! yes I was distracted

@CharlesAuthier please answer Antoine's comments, and then we'll open a new PR for the same branch.

**Pull request merged**